### DISCUSSION TITLE
Is different pictures based on product variants possible ?

### DISCUSSION THREAD

**Discussion by bennukem** [saleor/saleor#12312] [Q&A]
Is different pictures based on product variants possible ?

**Comment by usmanchaudhri** [comment]
https://docs.saleor.io/docs/3.x/api-reference/mutations/variant-media-assign

**Comment by timuric** [comment]
@bennukem you can upload all images on product level and assign specific images to variant level, please take a look at the dashboard

**Reply by Yahyaali1** [reply]
Is this still an option ? After uploading images I don't see any option to assign them to variants ?
